src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -246,8 +253,8 @@ public ListenableFuture<List<JobStatus>> getJobStatusesAsync(List<JobStatus> sta
         handleList(clazz, "job_statuses"));
   }
 
-  public List<Brand> getBrands() {
-    return complete(submit(req("GET", cnst("/brands.json")), handleList(Brand.class, "brands")));
+  public Iterable<Brand> getBrands() {
+    return new PagedIterable<>(cbp("/brands.json"), handleList(Brand.class, "brands"));
```

Comment:
This change doesn't match the current Rest API documentation.

According to it:

* a endpoint may or may not support CBP, I need to refer to the endpoint documentation (https://developer.zendesk.com/documentation/api-basics/pagination/paginating-through-lists-using-cursor-pagination/).
* refering to the brand endpoint, the only example I see uses OBP (https://developer.zendesk.com/api-reference/ticketing/account-configuration/brands/)

This is of importance, because according to the documentation, depending on the endpoint we need to check the endpoint doc to choose between "Paginating with the after cursor" and "Paginating with the next link".

Usually I would open a ticket with Zendesk support to get clarification and update of the doc if necessary, but in this case I trust you'll have a quicker answer by yourself :)

Reply:
@PierreBtz The support for Cursor Model is determined based on the endpoint use case and the depth of records we saw in the system for all customers. 

We are working on your feedback and currently evaluating the pagination information in the developer docs. We will get the right details added to every endpoint. There is active work in progress now on this outcome. Should be done soon.